src/injection/react-native-web.js

```diff
@@ -10,7 +10,7 @@ const {
   Dimensions,
   Easing,
 } = require('react-native-web');
-const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry');
+const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry').default;
```

Comment:
Well you're reaching into internals, so it's hardly a bug.
1. You could use import instead of require
2. I can add resolve back to StyleSheet